marklogic/src/main/scala/quasar/physical/marklogic/fs/package.scala

```diff
@@ -108,7 +109,7 @@ package object fs {
         rc.close.void
 
       def nextChunk(rc: ResultCursor) =
-        TV.map(rc.nextChunk)(xdmitem.toData)
+        TV.map(rc.nextChunk)(xdm => xdmitem.toData[ErrorMessages \/ ?](xdm) | Data.NA)
```

Comment:
No it's not. Just happened to notice it. Not a big deal if it's not done as part of this PR.